.changeset/chilly-spies-brush.md

```diff
@@ -0,0 +1,6 @@
+---
+'lit-html': patch
```

Comment:
Seems like this should be a minor bump, since it's adding a new API (albeit only in dev mode)?

Reply:
My thinking is that this is establishing a pattern that this is an unstable API that's free to change at any time, including in patch releases. On the other hand, that doesn't mean that we _have_ to only do so in patch releases. I think I find that convincing, will update.